web/src/components/apps/EmbeddedClusterManagement.tsx

```diff
@@ -457,50 +467,51 @@ const EmbeddedClusterManagement = ({
             />
           </div>
           <p className="tw-text-base tw-text-gray-600">
-            To add a node to this cluster, select the type of node you'd like to
-            add. Once you've selected a node type, we will generate a node join
-            command for you to use in the CLI. When the node successfully joins
-            the cluster, you will see it appear in the list of nodes on this
-            page.
+            {(rolesData?.roles || NODE_TYPES).length > 1 &&
```

Comment:
Won't `NODE_TYPES` always have a length of 1? If so, I think this should just be checking `rolesData?.roles`.

Reply:
yep, good call. will definitely make that more readable too